pkg/controllers/taint/clustertaintpolicy_controller.go

```diff
@@ -65,20 +66,21 @@ func (c *ClusterTaintPolicyController) Reconcile(ctx context.Context, req contro
 		return controllerruntime.Result{}, client.IgnoreNotFound(err)
 	}
 
-	if !clusterObj.DeletionTimestamp.IsZero() {
-		klog.V(4).Infof("Cluster(%s) is deleting.", req.Name)
-		return controllerruntime.Result{}, nil
-	}
```

Comment:
Why remove this?

Reply:
Because users may add some taints to the cluster that is being deleted, in this way, new applications cannot be scheduled to the cluster or existing applications can be evicted.